spec/models/measure_type_exclusion_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe MeasureTypeExclusion do
+  before do
+    allow(described_class).to receive(:exclusions).and_return(test_exclusions)
+  end
+
+  let(:test_exclusions) { {} }
+  let(:test_csv_file) { file_fixture('measure_type_exclusions.csv') }
+
+  describe '.load_from_file' do
+    subject { described_class.load_from_file(test_csv_file).exclusions }
+
+    it { is_expected.to eql(%w[735 1008] => %w[GB JE GG]) }
+  end
+
+  describe '.load_from_string' do
```

Comment:
How do you feel about killing this method and specs since we're not using it?

Reply:
Yeh, will do - put it in originally for testing - but then ended up adding a fixture to test the file loading anyway